notebooks/0-the-cloud/0-the-cloud.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # plt.subplot(projection = wcs) #TO FIX -this is not working, i think its something to do with my versions of matplotlib and astropy

</pre><p>It must be your version, I was able to plot with coordinates without any issues. I think we can un-comment this line and include it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>okay sounds good</p>